Framework/Kernel/test/ConfigObserverTest.h

```diff
@@ -0,0 +1,82 @@
+#ifndef MANTID_CONFIGOBSERVERTEST_H_
+#define MANTID_CONFIGOBSERVERTEST_H_
+
+#include "MantidKernel/ConfigService.h"
+#include "MantidKernel/ConfigObserver.h"
+#include "MantidKernel/System.h"
```

Comment:
This should not be necessary

Reply:
This is required for the `UNUSED_ARG` macro which suppresses the warnings for the [unused lambda parameters](https://github.com/mantidproject/mantid/blob/432eaaacd31284e198489f7fc300cccfe31b974a/Framework/Kernel/test/ConfigObserverTest.h#L51-L57).